server/main/templates/main/study-data.html

```diff
@@ -13,6 +13,8 @@
 {% block content %}
   {{ block.super }}
 
+  <h1>{% block body_title %}{{ block.super }}{% endblock body_title %}</h1>
```

Comment:
This does not make sense. A `block` templatetag that is overriding a parent template `block` should not be inside another `block`.

Reply:
I had conditionally set the title in the `block.super` depending on the presence of data, but it does make more sense to just set it directly here (and below).